src/nvim/mouse.c

```diff
@@ -966,6 +964,25 @@ bool do_mouse(oparg_T *oap, int c, int dir, int count, bool fixindent)
   return moved;
 }
 
+void handle_tabclose(int c1)
```

Comment:
use naming conventions that match surrounding code, if not the entire codebase
```suggestion
void do_tabclose(int c1)
```